### PR TITLE
Issue #63 cannot specify JSON Metadata Statement file paths

### PR DESCRIPTION
**Pull request by tnorimat**

### CODE REVIEW

File: webauthn4j-metadata/src/test/java/com/webauthn4j/metadata/JsonFileMetadataItemsProviderTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    void fetchMetadata() throws Exception {
+       List<Path> paths= new ArrayList<>(4);
+       paths.add(Paths.get(ClassLoader.getSystemResource("com/webauthn4j/metadata/JsonMetadataItem_fido2.json").toURI()));
+       paths.add(Paths.get(ClassLoader.getSystemResource("com/webauthn4j/metadata/JsonMetadataItem_u2f.json").toURI()));
+       paths.add(Paths.get(ClassLoader.getSystemResource("com/webauthn4j/metadata/JsonMetadataItem_uaf.json").toURI()));
+       paths.add(Paths.get(ClassLoader.getSystemResource("com/webauthn4j/metadata/JsonMetadataItem_unknown_protocol.json").toURI()));
+       JsonFileMetadataItemsProvider provider = new JsonFileMetadataItemsProvider(jsonConverter, paths);
+       readMetadataItem(provider.provide());
+       // read again
+       readMetadataItem(provider.provide());
```
Comment (ynojima) [inline webauthn4j-metadata/src/test/java/com/webauthn4j/metadata/JsonFileMetadataItemsProviderTest.java]: Please assert the same (cached) object is returned.

Comment (tnorimat) [reply webauthn4j-metadata/src/test/java/com/webauthn4j/metadata/JsonFileMetadataItemsProviderTest.java]: Would it mean that I need to check whether the object returned from second call of provider.provider() is the same one from first call in order to confirm that this test method run through all codes of JsonFileMetadataItemsProvider.provider() ?

Comment (ynojima) [reply webauthn4j-metadata/src/test/java/com/webauthn4j/metadata/JsonFileMetadataItemsProviderTest.java]: Yes

Comment (tnorimat) [reply webauthn4j-metadata/src/test/java/com/webauthn4j/metadata/JsonFileMetadataItemsProviderTest.java]: So sorry...

**Pull request merged**